tools/explorer/tt_adapter/src/tt_adapter/runner.py

```diff
@@ -131,8 +135,31 @@ def run(
             "DEVICE FW DURATION [ns]",
             "CORE COUNT",
             "OUTPUT_0_MEMORY",
+            "LOC",
         ]
         perf = perf[columns]
-        print(perf)
 
-        print("Total device duration: ", perf["DEVICE FW DURATION [ns]"].sum(), "ns")
+        # Create the node_data type here
+        timing_data = list(zip(perf["LOC"], perf["DEVICE FW DURATION [ns]"]))
```

Comment:
So do we have this data in ttrt on main?

Reply:
Yeah, this was merged into main yesterday.